### PR TITLE
Update user guide for `DeletePersonFromModuleCommand`

### PR DESCRIPTION
**Pull request by ekweirui**
Currently, the user guide does not contain information for `DeletePersonFromModuleCommand`.

Thus, this pull request does the following:
* Update user guide for `DeletePersonFromModuleCommand`

### CODE REVIEW

**Review by cheeheng — DISMISSED** [review]
LGTM

**Review by teoyuqi — DISMISSED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -34,27 +34,27 @@ contains some sample data.<br>
...
+| [`delete-module`](#212-delete-module)               | `delete-module   m/MODULE_CODE`                                         | Delete module by module code                                                    |
+| [`edit-module`](#213-edit-module)                   | `edit-module     INDEX ([m/MODULE_CODE] [t/MODULE_TITLE])`              | Edit module belonging to the specified index currently displayed on the screen  |
+| *[`find-module`](#214-find-module)                  | `find-module     KEYWORD`                                               | Find module that starts with specified keyword in home page                     |
+| *[`list-module`](#215-list-module)                  | `list-module`                                                           | List all modules in home page after finding                                     |
+| [`add-person-to-module`](#216-add-person-to-module) | `add-person-to-module m/MODULE_CODE <br/>n/NAME`                        | Add person with specified name to the module with the specified module code     |
```
Comment (cheeheng) [inline docs/UserGuide.md:44]: Oops, looks like you accidentally deleted your section from this table!

**Review by teoyuqi — APPROVED** [review]
LGTM

**Review by cheeheng — COMMENTED** [review]
LGTM except that one command seems to be missing in the command summary.

File: docs/UserGuide.md (hunk 1 above)
Comment (ekweirui) [reply docs/UserGuide.md:44]: Yes sorry I was editing on my phone! If I were to edit back I would need about 1 hour from now as I’m currently outside. If that is too long, perhaps you could merge this in and help me to add it in? 🙏

**Review by cheeheng — APPROVED** [review]
LGTM
Will fix missing column issue for @ekweirui

**Pull request merged**